### ISSUE TITLE
Consider migrating to GitHub Action

### ISSUE THREAD

**Issue by peter279k** [Xantios/mimey#4]
As title, since the Travis CI org will be deprecated soon, the Travis CI com has been limited for the open source preojects.

I think it's time to consider migrating to the GitHub Actions.

Once this issue has been accepted by repository owner, I will be happy to work on that.

**Comment by Xantios** [comment]
Truth be told, i only inherited the travis file from the original repository and set it up so its compatible as a drop in replacement for all that want to work on it.

Correct me if I'm wrong, but if i look at travis CI (the dot com website) the pricing is still free for open source.
and using GitHub Actions for me personally feels like putting all my eggs in one basket, which can arguably be a good thing if you want it to integrate well however it can also be a hell to migrate if Github changes their ways

I do agree that if TravisCI is going to change their model in a way that directly has consequences for this package we should consider alternatives

So long story short, i am not well informed enough to make this decision on my own right now. please elaborate a bit on why you think we should move it and maybe add some alternative so the community can pick what they think is best.

**Comment by peter279k** [comment]
@Xantios, I think it's good to migrate to Travis CI dot com.

**Comment by peter279k** [comment]
@Xantios, just notice that you should migrate Travis CI dot org to the Travis CI dot com.
